### PR TITLE
changelog for release 0.81.0

### PR DESCRIPTION
**Pull request by atoulme**

### CODE REVIEW

**Comment by atoulme** [comment]
@jeffreyc-splunk yes of course. Thanks, added those.

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +23,54 @@
...
+- (Contrib) `sqlqueryreceiver`: Add support of Start and End Timestamp Column in Metric Configuration. ([#18925](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/18925), [#14146](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/14146))
+- (Contrib) `filelogreceiver`: Add support for tracking the current file in filelogreceiver ([#22998](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/22998))
+- (Contrib) `pkg/ottl`: Adds new `Time` converter to convert a string to a Golang time struct based on a supplied format ([#22007](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/22007))
+- (Contrib) `hostmetricsreceiver`: Add new Windows-exclusive process.handles metric. ([#21379](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21379))
+- `resourcedetectionprocessor`: Adds a way to configure the list of added resource attributes by the processor ([#21482](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21482))
```
Comment (jeffreyc-splunk) [inline CHANGELOG.md:36]:
```suggestion
- (Contrib) `resourcedetectionprocessor`: Adds a way to configure the list of added resource attributes by the processor ([#21482](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21482))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +23,54 @@
...
+          enabled: true
+  ```
+
+  For example, this config makes `host.name` being set by `system` detector, and `host.id` by `ec2` detector.
+  Moreover:
+  - Existing behavior remains unaffected as all attributes are currently enabled by default.
+  - The default attributes 'enabled' values are defined in `metadata.yaml`.
+  - Future releases will introduce changes to resource_attributes `enabled` values.
+  - Users can tailor resource detection process to their needs and environment.
+- `k8sclusterreceiver`: Switch k8s.pod and k8s.container metrics to use pdata. ([#23441](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/23441))
```
Comment (jeffreyc-splunk) [inline CHANGELOG.md:63]:
```suggestion
- (Contrib) `k8sclusterreceiver`: Switch k8s.pod and k8s.container metrics to use pdata. ([#23441](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/23441))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +23,54 @@
...
+
+### 🧰 Bug fixes 🧰
+
+- `k8sclusterreceiver`: Add back all other vendor-specific node conditions, and report them even if missing, as well as all allocatable node metrics if present,  to the list of Kubernetes node metrics available, which went missing during the pdata translation ([#23839](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/23839))
```
Comment (jeffreyc-splunk) [inline CHANGELOG.md:67]:
```suggestion
- (Contrib) `k8sclusterreceiver`: Add back all other vendor-specific node conditions, and report them even if missing, as well as all allocatable node metrics if present,  to the list of Kubernetes node metrics available, which went missing during the pdata translation ([#23839](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/23839))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -11,6 +23,54 @@
...
+- `k8sclusterreceiver`: Add explicitly `k8s.node.allocatable_pods` to the list of Kubernetes node metrics available, which went missing during the pdata translation ([#23839](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/23839))
```
Comment (jeffreyc-splunk) [inline CHANGELOG.md:68]:
```suggestion
- (Contrib) `k8sclusterreceiver`: Add explicitly `k8s.node.allocatable_pods` to the list of Kubernetes node metrics available, which went missing during the pdata translation ([#23839](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/23839))
```

**Review by theletterf — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -11,6 +23,54 @@
...
+- `receiver/kafkametricsreceiver`: Updates certain metrics in kafkametricsreceiver to function as non-monotonic sums. ([#4327](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/4327))
```
Comment (jeffreyc-splunk) [inline CHANGELOG.md:69]:
```suggestion
- (Contrib) `receiver/kafkametricsreceiver`: Updates certain metrics in kafkametricsreceiver to function as non-monotonic sums. ([#4327](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/4327))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -11,6 +23,54 @@
...
+  Update the metrics type in KafkaMetricsReceiver from "gauge" to "nonmonotonic sum". Changes metrics are, kafka.brokers, kafka.topic.partitions, kafka.partition.replicas, kafka.partition.replicas_in_sync, kafka.consumer_group.members.
+- `vcenterreceiver`: Fixed a bug in which the vCenter receiver was incorrectly emitting the "used" attribute twice under the vcenter.datastore.disk.usage metric instead of "used" and "available". ([#23654](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/23654))
```
Comment (jeffreyc-splunk) [inline CHANGELOG.md:71]: Should we exclude this since we don't include the vcenter receiver?

File: CHANGELOG.md [hunk 7]
```diff
@@ -11,6 +23,54 @@
...
+- `windowseventlogreceiver`: Fix buffer overflow when ingesting large raw Events ([#23677](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/23677))
```
Comment (jeffreyc-splunk) [inline CHANGELOG.md:72]:
```suggestion
- (Contrib) `windowseventlogreceiver`: Fix buffer overflow when ingesting large raw Events ([#23677](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/23677))
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -11,6 +23,54 @@
...
+- `pkg/stanza`: adding octet counting event breaking for syslog parser ([#23577](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/23577))
```
Comment (jeffreyc-splunk) [inline CHANGELOG.md:73]:
```suggestion
- (Contrib) `pkg/stanza`: adding octet counting event breaking for syslog parser ([#23577](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/23577))
```

File: CHANGELOG.md (hunk 6 above)
Comment (atoulme) [reply CHANGELOG.md:71]: Sorry, I missed that.

Comment (atoulme) [reply CHANGELOG.md:71]:
```suggestion
```

**Review by jeffreyc-splunk — APPROVED** [review]

**Review by rmfitzpatrick — APPROVED** [review]

**Pull request merged**